src/link.ts

```diff
@@ -486,7 +490,7 @@ export class LinkContainer {
     idTypeFilter : <(idtype: IDType, i: number, dataVis: IDataVis) => boolean>constantTrue,
```

Comment:
I create a local function called `constantTrue` independent of the core. It happens to have the same name as the deprecated method. There is no deeper logic behind it.